src/lambda/local/detectLocalLambdas.ts

```diff
@@ -17,6 +17,7 @@ export interface LocalLambda {
     resource: CloudFormation.Resource
     templatePath: string
     handler?: string
+    codeUri?: string
```

Comment:
It looks like this `codeUri` field is directly related to `resource`. Is there any reason that code needing this field can't look it up from the `resource` field?

`getCodeUri` would become a convenience method within cloudFormation.ts

Reply:
✔️